fluid/StyleParse.cxx

```diff
@@ -0,0 +1,325 @@
+//
+// Syntax highlighting style parser class - erco 09/16/2020
```

Comment:
Standard header would read "... for the Fast Light Tool Kit (FLTK)" or similar. I'd prefer if you put your "erco ..." in another line (formalism, nitpicking, sorry).

Reply:
OK, for consistency with existing headers I can use what we used in Fl_Table/Fl_Tree, which is to use the author's name in the copyright header, e.g.

```
//
// Fl_Table -- A table widget for the Fast Light Tool Kit (FLTK).
//
// Copyright 2002 by Greg Ercolano.
// Copyright (c) 2004 O'ksi'D
//
`

Also, I'll try to remember the "for the Fast Light Tool Kit (FLTK)." suffix; I wasn't aware that was a standard thing, but it seems to be.